tests/TestHelpers/EmailHelper.php

```diff
@@ -26,238 +26,53 @@
-		?int $waitTimeSec = EMAIL_WAIT_TIMEOUT_SEC
-	) {
-		$currentTime = \time();
-		$endTime = $currentTime + $waitTimeSec;
-
-		while ($currentTime <= $endTime) {
-			$skip = 1;
-			foreach (self::getEmails($localMailhogUrl, $xRequestId)->items as $item) {
-				$thisEmailAddress
-					= $item->To[0]->Mailbox . "@" . $item->To[0]->Domain;
-				if ($thisEmailAddress === $emailAddress) {
-					if ($skip < $emailNumber) {
-						$skip++;
-						continue;
-					}
-					return $item->From->Mailbox . '@' . $item->From->Domain;
-				}
-			}
-			\usleep(STANDARD_SLEEP_TIME_MICROSEC * 50);
-			$currentTime = \time();
-		}
-		throw new Exception("Could not find the email to the address: " . $emailAddress);
-	}
-
-	/**
-	 *
-	 * @param string|null $localMailhogUrl
-	 * @param string|null $xRequestId
-	 *
-	 * @return ResponseInterface
-	 * @throws GuzzleException
-	 */
-	public static function deleteAllMessages(
-		?string $localMailhogUrl,
-		?string $xRequestId
-	):ResponseInterface {
-		return HttpRequestHelper::delete(
-			$localMailhogUrl . "/api/v1/messages",
-			$xRequestId
-		);
-	}
 
 	/**
-	 * Returns the host name or address of the Mailhog server as seen from the
+	 * Returns the host name or address of the Email server as seen from the
 	 * point of view of the system-under-test.
 	 *
 	 * @return string
 	 */
-	public static function getMailhogHost():string {
-		$mailhogHost = \getenv('MAILHOG_HOST');
-		if ($mailhogHost === false) {
-			$mailhogHost = "127.0.0.1";
+	public static function getEmailHost():string {
+		$emailHost = \getenv('EMAIL_HOST');
+		if ($emailHost === false) {
+			$emailHost = "127.0.0.1";
 		}
-		return $mailhogHost;
+		return $emailHost;
 	}
 
 	/**
-	 * Returns the host name or address of the Mailhog server as seen from the
+	 * Returns the host name or address of the Email server as seen from the
 	 * point of view of the test runner.
 	 *
 	 * @return string
 	 */
-	public static function getLocalMailhogHost():string {
-		$localMailhogHost = \getenv('LOCAL_MAILHOG_HOST');
-		if ($localMailhogHost === false) {
-			$localMailhogHost = self::getMailhogHost();
+	public static function getLocalEmailHost():string {
+		$localEmailHost = \getenv('LOCAL_EMAIL_HOST');
+		if ($localEmailHost === false) {
+			$localEmailHost = self::getEmailHost();
 		}
-		return $localMailhogHost;
+		return $localEmailHost;
 	}
 
 	/**
-	 * Returns the host and port where Mailhog messages can be read and deleted
+	 * Returns the host and port where Email messages can be read and deleted
 	 * by the test runner.
 	 *
 	 * @return string
 	 */
-	public static function getLocalMailhogUrl():string {
-		$localMailhogHost = self::getLocalMailhogHost();
+	public static function getLocalEmailUrl():string {
+		$localEmailHost = self::getLocalEmailHost();
 
-		$mailhogPort = \getenv('MAILHOG_PORT');
-		if ($mailhogPort === false) {
-			$mailhogPort = "8025";
+		$emailPort = \getenv('EMAIL_PORT');
+		if ($emailPort === false) {
+			$emailPort = "9000";
```

Comment:
env vars will be treated as string
```suggestion
		$emailPort = \getenv('EMAIL_PORT');
		if ($emailPort === '' || $emailPort === 'false') {
			$emailPort = "9000";
```

Reply:
func `getenv` return boolean(`false`) if the env is not set